src/controllers/agent-controller.js

```diff
@@ -114,20 +115,20 @@ const putImageSnapshotEndPoint = async function (req, fog) {
-  getAgentMicroservicesEndPoint: AuthDecorator.checkFogToken(getAgentMicroservicesEndPoint),
-  getAgentMicroserviceEndPoint: AuthDecorator.checkFogToken(getAgentMicroserviceEndPoint),
-  getAgentRegistriesEndPoint: AuthDecorator.checkFogToken(getAgentRegistriesEndPoint),
-  getAgentTunnelEndPoint: AuthDecorator.checkFogToken(getAgentTunnelEndPoint),
-  getAgentStraceEndPoint: AuthDecorator.checkFogToken(getAgentStraceEndPoint),
-  updateAgentStraceEndPoint: AuthDecorator.checkFogToken(updateAgentStraceEndPoint),
-  getAgentChangeVersionCommandEndPoint: AuthDecorator.checkFogToken(getAgentChangeVersionCommandEndPoint),
-  updateHalHardwareInfoEndPoint: AuthDecorator.checkFogToken(updateHalHardwareInfoEndPoint),
-  updateHalUsbInfoEndPoint: AuthDecorator.checkFogToken(updateHalUsbInfoEndPoint),
-  deleteNodeEndPoint: AuthDecorator.checkFogToken(deleteNodeEndPoint),
-  getImageSnapshotEndPoint: AuthDecorator.checkFogToken(getImageSnapshotEndPoint),
-  putImageSnapshotEndPoint: AuthDecorator.checkFogToken(putImageSnapshotEndPoint)
+  getAgentConfigEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentConfigEndPoint)),
+  updateAgentConfigEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(updateAgentConfigEndPoint)),
+  getAgentConfigChangesEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentConfigChangesEndPoint)),
+  updateAgentStatusEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(updateAgentStatusEndPoint)),
+  getAgentMicroservicesEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentMicroservicesEndPoint)),
+  getAgentMicroserviceEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentMicroserviceEndPoint)),
+  getAgentRegistriesEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentRegistriesEndPoint)),
+  getAgentTunnelEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentTunnelEndPoint)),
+  getAgentStraceEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentStraceEndPoint)),
+  updateAgentStraceEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(updateAgentStraceEndPoint)),
+  getAgentChangeVersionCommandEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getAgentChangeVersionCommandEndPoint)),
+  updateHalHardwareInfoEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(updateHalHardwareInfoEndPoint)),
+  updateHalUsbInfoEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(updateHalUsbInfoEndPoint)),
+  deleteNodeEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(deleteNodeEndPoint)),
+  getImageSnapshotEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(getImageSnapshotEndPoint)),
+  putImageSnapshotEndPoint: AuthDecorator.checkFogToken(AgentDecorator.updateLastActive(putImageSnapshotEndPoint))
 };
```

Comment:
how about adding this logic to checkFogToken method of AuthDecorator?
There exported functions decorated multiple times look pretty ugly